BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -147,6 +147,184 @@ public static Vector3 GetAirToGroundTarget(Vector3 targetPosition, Vector3 targe
             return targetPosition + (targetVelocity * leadTime);
         }
 
+        public static Vector3 GetAirToAirLoftTarget(Vector3 targetPosition, Vector3 targetVelocity,
+            Vector3 targetAcceleration, Vessel missileVessel, float targetAlt, float maxAltitude,
+            float rangeFactor, float altComp, float velComp, float loftAngle, float termAngle,
+            float termDist, ref int loftState, out float timeToImpact, out float targetDistance,
+            float minSpeed = 200)
+        {
+
+            Vector3 velDirection = missileVessel.Velocity().normalized;
+
+            targetDistance = Vector3.Distance(targetPosition, missileVessel.transform.position);
+
+            float currSpeed = Mathf.Max((float)missileVessel.srfSpeed, minSpeed);
+            Vector3 currVel = currSpeed * velDirection;
+
+            //Vector3 Rdir = (targetPosition - missileVessel.transform.position).normalized;
+            //float rDot = Vector3.Dot(targetVelocity - currVel, Rdir);
+
+            float leadTime = (float)(1 / ((targetVelocity - currVel).magnitude / targetDistance));
+            //float leadTime = (targetDistance / rDot);
+
+            timeToImpact = leadTime;
+            leadTime = Mathf.Clamp(leadTime, 0f, 8f);
+
+            // If loft is not terminal
+            if ((targetDistance > termDist) && (loftState < 3))
+            {
+                if (BDArmorySettings.DEBUG_MISSILES) Debug.Log("[BDArmory.MissileLauncher]: Lofting");
```

Comment:
`[BDArmory.MissileLauncher]:` should be `[BDArmory.MissileGuidance]:` since it's in the MissileGuidance.cs file.
Also in several places below.

Reply:
Replaced